UPGRADE.md

```diff
@@ -1,6 +1,12 @@
 UPGRADE
 =======
 
+## From 1.0.0-alpha to 1.0.0-beta
```

Comment:
We don't have `alpha` release yet. This should be in "From 0.19 to 1.0.0-alpha" section :)